Sources/SwiftBeanCountModel/Commodity.swift

```diff
@@ -46,6 +46,23 @@ public class Commodity {
         return .valid
     }
 
+    /// Validates the commodity in the context of enabled plugins
```

Comment:
please follow the same format for parameters as validate on transactions: (in: Ledger).

Also, this should replace the old validate function